--- releven_people.rq	2025-08-20 12:01:24.609173100 +0200
+++ releven_people.rq.changed	2025-08-20 12:03:03.223059700 +0200
@@ -146,6 +146,6 @@
   } GROUP BY ?People }
   BIND (COALESCE(?P_person_possession_assertion_count, 0) AS ?P_person_possession_assertion)
 
-  BIND (?People_person_gender_assignment + ?People_person_ethnic_group_membership_assertion + ?People_person_population_membership_assertion + ?People_person_part_of_manifest_group_assertion + ?People_person_social_relationship + ?People_person_language_skill + ?People_person_legal_role + ?People_person_religious_affiliation + ?People_person_possession_assertion AS ?n_assertions)
+  BIND (?P_person_gender_assignment + ?P_person_ethnic_group_membership_assertion + ?P_person_population_membership_assertion + ?P_person_part_of_manifest_group_assertion + ?P_person_social_relationship + ?P_person_language_skill + ?P_person_legal_role + ?P_person_religious_affiliation + ?P_person_possession_assertion AS ?n_assertions)
 
 }
