--- releven_places.py	2025-11-03 12:55:37.295148012 +0100
+++ releven_places.py.changed	2025-11-03 12:55:45.475187261 +0100
@@ -37,8 +37,4 @@
 
     place_display_name: Annotated[str | None, SPARQLBinding("Places_place_display_name")]
     place_id_assignment: Annotated[list[IdentityInOtherServices], SPARQLBinding("Places_place_id_assignment")]
-    place_name_of_place_assertion: Annotated[int, SPARQLBinding("Places_place_name_of_place_assertion")]
-    place_part_of_place_assertion: Annotated[int, SPARQLBinding("Places_place_part_of_place_assertion")]
-    place_type_of_place_assertion: Annotated[int, SPARQLBinding("Places_place_type_of_place_assertion")]
-    place_succeeded_by_assertion: Annotated[int, SPARQLBinding("Places_place_succeeded_by_assertion")]
-    place_had_population_assertion: Annotated[int, SPARQLBinding("Places_place_had_population_assertion")]
+    n_assertions: int
