--- releven_text.py	2025-11-06 11:59:23.776268540 +0100
+++ releven_text.py.changed	2025-11-06 11:59:22.396267356 +0100
@@ -3,20 +3,6 @@
 from typing import Annotated
 
 
-
-class Creation(BaseModel):
-    model_config = ConfigDict(
-        title="",
-    )
-
-    written_text_creation_time_frame_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_creation_written_text_creation_time_frame_assertion")]
-    written_text_creation_author_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_creation_written_text_creation_author_assertion")]
-    written_text_creation_place_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_creation_written_text_creation_place_assertion")]
-    written_text_creation_copied_from_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_creation_written_text_creation_copied_from_assertion")]
-    written_text_creation_translated_from_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_creation_written_text_creation_translated_from_assertion")]
-
-
-
 class Text(BaseModel):
     model_config = ConfigDict(
         title="",
@@ -24,8 +10,4 @@
     id: Annotated[AnyUrl, SPARQLBinding("Text")]
 
     written_text_display_name: Annotated[str | None, SPARQLBinding("Text_written_text_display_name")]
-    written_text_title_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_title_assertion")]
-    written_text_creation: Annotated[Creation | None, SPARQLBinding("Text_written_text_creation")] = None
-    written_text_published_as_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_published_as_assertion")]
-    written_text_written_in_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_written_in_assertion")]
-    written_text_version_of_assertion: Annotated[int | None, SPARQLBinding("Text_written_text_version_of_assertion")]
+    n_assertions: int
